messaging-activemq/src/test/resources/org/wildfly/extension/messaging/activemq/subsystem_13_0.xml

```diff
@@ -0,0 +1,719 @@
+                sync-transactional="${journal.sync.transactional:false}"
+                sync-non-transactional="${journal.sync.non.transactional:true}"
+                log-write-rate="${log.journal.write.rate:true}"
+                file-size="${journal.file.size:102400}"
+                min-files="${journal.min.files:2}"
+                pool-files="${journal.pool.files:5}"
+                compact-percentage="${journal.compact.percentage:83}"
+                file-open-timeout="${journal.file.open.timeout:7}"
+                compact-min-files="${journal.compact.min.files:2}"
+                max-io="${journal.max.io:50}"
+                create-bindings-dir="${create.bindings.dir:false}"
+                create-journal-dir="${create.journal.dir:true}"
+                datasource="fooDS"
+                database="mysql"
+                jdbc-lock-expiration="${journal.jdbc.lock.expiration:891}"
+                jdbc-lock-renew-period="${journal.jdbc.lock.renew.period:892}"
+                jdbc-network-timeout="${journal.jdbc.network.timeout:4567}"
+                messages-table="${journal.message.table:MESSAGES}"
+                bindings-table="${journal.bindings.table:BINDINGS}"
+                jms-bindings-table="${journal.jms.bindings.table:JMS_BINDINGS}"
+                large-messages-table="${journal.large-messages.table:LARGE_MESSAGES}"
+                page-store-table="${journal.page-store.table:PAGE_STORE}"
+                node-manager-store-table="${journal.node-manager-store-table:NODE_MANAGER_STORE}" />
+
+        <statistics
+            enabled="${statistics.enabled:true}"
+            message-counter-sample-period="${message.counter.sample.period:7654}"
+            message-counter-max-day-history="${message.counter.max.day.history:23}"/>
+
+        <transaction
+            timeout="${transaction.timeout:4321}"
+            scan-period="${transaction.timeout.scan.period:5432}"/>
+
+        <message-expiry
+            scan-period="${message.expiry.scan.period:7654}"
+            thread-priority="${message.expiry.thread.priority:7}"/>
+
+        <debug
+            perf-blast-pages="${perf.blast.pages:70}"
+            run-sync-speed-test="${run.sync.speed.test:false}"
+            server-dump-interval="${server.dump.interval:8642}"
+            memory-warning-threshold="${memory.warning.threshold:1024}"
+            memory-measure-interval="${memory.measure.interval:2048}"/>
+
+        <network-isolation
+            network-check-list="127.0.0.1,[::1]"
+            network-check-nic="localhost"
+            network-check-period="5000"
+            network-check-ping-command="ping -c 1 -t %d %s"
+            network-check-ping6-command="ping6 -c 1 %2$s"
+            network-check-timeout="1000"
+            network-check-url-list="127.0.0.1"/>
+
+        <critical-analyzer
+            enabled="true"
+            check-period="0"
+            timeout="120000"
+            policy="LOG"/>
```

Comment:
Please use an expression for all attributes that support it. Network isolation section too.

Reply:
@ehsavoie Your resolved this comment but I don't see any change.